File: python/23d_581A_VasyaTheHipster.py
def main():
    a, b = input().split()
    a, b = int(a), int(b)
    dif_sock = min(a,b)
    same_sock = (max(a,b) - dif_sock) // 2
    print(dif_sock, same_sock)

if __name__ == "__main__":
    main()

File: cpp/23d_581A_VasyaTheHipster.cpp
#include<bits/stdc++.h>

int main(){
    int a, b, dif_sock, same_sock;
    std::cin>>a>>b;
    dif_sock = std::min(a,b);
    same_sock = (std::max(a,b) - dif_sock) / 2;
    std::cout<<dif_sock<<" "<<same_sock;
    return 0;
}